src/One/BitbucketProvider.php

```diff
@@ -0,0 +1,7 @@
+<?php namespace Laravel\Socialite\One;
+
+class BitbucketProvider extends AbstractProvider
+{
+
```

Comment:
I know :( just following this package conventions https://github.com/laravel/socialite/blob/2.0/src/One/TwitterProvider.php